src/pages/[platform]/build-a-backend/storage/copy-files/index.mdx

```diff
@@ -66,7 +66,41 @@ Cross identity ID copying is only allowed if the destination path has the the ri
 
 </Callout>
 
-### More `copy` options
+Alternatively, specify a non-default bucket target by including `bucket` in both the source and destination objects. 
+
+See [Configure additional storage buckets](/[platform]/build-a-backend/storage/set-up-storage/#configure-additional-storage-buckets) for more information.
+
+```javascript
+import { copy } from 'aws-amplify/storage';
+
+const copyFile = async () => {
+  try {
+    const response = await copy({
+      source: {
+        path: 'album/2024/1.jpg',
+        // Alternatively, path: ({identityId}) => `album/{identityId}/1.jpg`
+        bucket: "userDefinedFriendlyBucketName"
+      },
+      destination: {
+        path: 'shared/2024/1.jpg',
+        // Alternatively, path: ({identityId}) => `shared/{identityId}/1.jpg`
+        bucket: {
+          bucketName: "generated-secondary-bucket-name",
+          region: "us-east-2"
+        }
+      },
+    });
+  } catch (error) {
+    console.error('Error', err);
+  }
+};
+```
+
+<Callout>
+In order to copy to or from non-default buckets, both source and destination must have `bucket` explicitly defined.
+</Callout>
+
+## More `copy` options
```

Comment:
we would still want to include bucket as option in the table with the types specified as `string` | `{bucketName: string; region:string}`

Reply:
It felt awkward including bucket in the table here because the only existing option is `useAccelerateEndpoint` which is a backend option and the copy operation, source, and destination don't take an object labeled `options` like some of our other APIs. This seems like it might be confusing to customers and worth calling out.

It might be a better idea to split these into two different tables? One to identify the backend option and another to document the available options to the source and destination inputs.

Reply:
We will remove this section alltogether since there is no options